decidim/decidim
# Fix newsletter subscriber count

#### :tophat: What? Why?
while reviewing #13429 , i have noticed there is a javascript error in the page. 
On a closer look, i have found that the respective javascript is related to the recipients counter, which was not updated. 

![image](https://github.com/user-attachments/assets/540148be-9350-412b-9dff-21db49d9be20)

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13429

#### Testing
1. Login as admin , and create a newsletter 
2. Go to "Select recipients to deliver "
3. In that screen see how many users it indicates that you will send the newsletter
4. Start selecting Participants or Followers and some spaces ( see also the js error in console)
5. See the counter is not changing
6. Apply the patch 
7. Repeat 3,4
8. See the counter change it's value and js error is gone

:hearts: Thank you!
